build.gradle

```diff
@@ -0,0 +1,133 @@
+buildscript {
+    dependencies {
+        classpath files('gradle/gradle-witness.jar')
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'witness'
+apply plugin: 'maven-publish'
+apply plugin: 'application'
+
+compileJava {
+    sourceCompatibility = 1.7
+    targetCompatibility = 1.7
+}
+
+version = "37"
+
+repositories {
+    mavenLocal()
+    maven { url "http://4thline.org/m2" }
+    maven { url 'https://mvn.freenetproject.org' }
+    jcenter()
+}
+
+configurations {
+    extraLibs
+}
+
+dependencies {
+    compile group: 'org.freenetproject', name: 'fred', version: 'build+'
+    compile group: 'org.yaml', name: 'snakeyaml', version: '1.5'
+    testImplementation 'junit:junit:4.13'
+}
+
+def gitrev
+task buildInfo {
+    try {
+        def cmd = "git describe --always --abbrev=4 --dirty"
+        def proc = cmd.execute()
+        gitrev = proc.text.trim()
+    } catch (java.io.IOException e) {
+        gitrev = "@unknown@"
+    }
+}
+
+test {
+	useJUnit()
+}
+
+application {
+    mainClassName = 'plugins.Library.ui.TestInterface'
+}
+
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.attribute.BasicFileAttributes;
+
+Date getMTime(String file) {
+    return new Date(Files.readAttributes(Paths.get(file), BasicFileAttributes.class).lastModifiedTime().toMillis());
+}
+
+jar {
+    manifest {
+        attributes 'Plugin-Main-Class': 'plugins.Library.Main',
+                   'Main-Class': 'plugins.Library.ui.TestInterface',
+                   'Required-Node-Version': '1239',
+                   'Implementation-Version': version,
+                   'Built-By': System.getProperty('user.name'),
+                   'Built-Date': getMTime("src/main/java/plugins/Library/Version.java"),
+                   'Built-JDK': System.getProperty('java.version')
+    }
+    from('src') {
```

Comment:
Everything from the resources directory is included automatically.

Reply:
True. Done! :+1: